<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShowCommandDialog</class>
 <widget class="QDialog" name="ShowCommandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QTableWidget" name="tableWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>120</width>
       <height>16777215</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>time</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Frequency:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="frequencySpin">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tableButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="arrow-left">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QToolButton" name="listAddButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="listRemoveButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="listRefreshButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="backgroundBrush">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </property>
     <property name="sceneRect">
      <rectf>
       <x>0.000000000000000</x>
       <y>0.000000000000000</y>
       <width>0.000000000000000</width>
       <height>0.000000000000000</height>
      </rectf>
     </property>
     <property name="renderHints">
      <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::TextAntialiasing</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ShowCommandDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ShowCommandDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
